README.md

```diff
@@ -131,6 +131,13 @@ the following line to your `Podfile`:
     pod 'gRPC-Swift-Plugins'
 ```
 
+#### Homebrew
+
+The plugins are available from [homebrew](https://brew.sh) and can be installed with:
+```bash
+    $ brew install swift-protobuf grpc-swift
+```
+
 The plugins are available in the `Pods/gRPC-Swift-Plugins/` folder afterwards.
```

Comment:
Ah, this line has escaped from the section above, it should come before the `#### Homebrew` line.

Reply:
Yup
You are right!

Done